types/signals/index.d.ts

```diff
@@ -3,38 +3,38 @@
 // Definitions by: Diullei Gomes <https://github.com/diullei>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare var signals: signals.SignalWrapper;
+declare var signals: signals.SignalWrapper<any>;
 
 export = signals;
 export as namespace signals;
 
 declare namespace signals {
 
-    interface SignalWrapper {
-        Signal: Signal
+    interface SignalWrapper<T> {
+        Signal: Signal<T>
     }
 
-    interface SignalBinding {
+    interface SignalBinding<T> {
         active: boolean;
         context: any;
         params: any;
         detach(): Function;
         execute(paramsArr?: any[]): any;
-        getListener(): Function;
-        getSignal(): Signal;
+        getListener(): (...params: T[]) => void;
+        getSignal(): Signal<T>;
         isBound(): boolean;
         isOnce(): boolean;
     }
 
-    interface Signal {
+    interface Signal<T> {
```

Comment:
Maybe this should be `Signal<T = any>` to avoid breaking other packages (see travis)?

Reply:
Thanks ! I was wondering how to get this working on Travis 😄